test/autorest/urlgroup/queries_test.go

```diff
@@ -103,6 +273,27 @@ func TestGetBooleanTrue(t *testing.T) {
 	helpers.VerifyStatusCode(t, result, http.StatusOK)
 }
 
+// GetIntNegativeOneMillion - Get '-1000000' integer value
+func TestGetIntNegativeOneMillion(t *testing.T) {
+	client := getQueriesClient(t)
+	result, err := client.GetIntNegativeOneMillion(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+	helpers.VerifyStatusCode(t, result, http.StatusOK)
```

Comment:
Wondering if this should have a deepequal check for the value?

Reply:
Same for several of these tests that do GETs.

Reply:
This one doesn't return a model, just an `*http.Resposne` (looks like they all do)..